Scripts/onscrlib.sh

```diff
@@ -65,7 +65,7 @@ export PATH="/opt/local/bin:/opt/local/sbin:/usr/local/bin/:/usr/local/sbin/:$PA
 
 # Fixes bz2 unpack issues
 function tar() {
-  /usr/bin/tar "$@"
+  /opt/local/bin/gnutar "$@"
```

Comment:
Please use normal tar on macOS. Not GNU variant. Clarify why do you need gnu tar otherwise, so that we can find a solution.

Reply:
It was because I believe libepoxy has no upper folder (it fills up the src folder)
It is probably easier to just change the libepoxy extract() function

Reply:
Use https://download.gnome.org/sources/libepoxy/1.5/libepoxy-1.5.10.tar.xz